Makefile

```diff
@@ -517,7 +517,7 @@ bench-idle:
 	$(NODE) benchmark/idle_clients.js &
 
 jslint:
-	$(NODE) tools/eslint/bin/eslint.js src lib test tools/eslint-rules \
+	$(NODE) tools/eslint/bin/eslint.js lib src test tools/doc tools/eslint-rules \
```

Comment:
You also need to update the corresponding line in [vcbuild.bat](https://github.com/nodejs/node/blob/master/vcbuild.bat#L262)